tests/integration/integration_nodejs_test.go

```diff
@@ -489,6 +489,44 @@ func TestProviderSecretConfig(t *testing.T) {
 	})
 }
 
+func TestResourceWithSecretSerializationNodejs(t *testing.T) {
+	integration.ProgramTest(t, &integration.ProgramTestOptions{
+		Dir:          filepath.Join("secret_outputs", "nodejs"),
+		Dependencies: []string{"@pulumi/pulumi"},
+		Quick:        true,
+		ExtraRuntimeValidation: func(t *testing.T, stackInfo integration.RuntimeValidationStackInfo) {
+			// The program exports three resources:
+			//   1. One named `withSecret` who's prefix property should be secret, specified via `pulumi.secret()`.
+			//   2. One named `withSecretAdditional` who's prefix property should be a secret, specified via
+			//      additionalSecretOutputs.
+			//   3. One named `withoutSecret` which should not be a secret.
+			// We serialize both of the these as POJO objects, so they appear as maps in the output.
```

Comment:
What's a POJO?

Reply:
I was tying to find a good definition. https://masteringjs.io/tutorials/fundamentals/pojo is prob a good summary.

Reply:
aha thank you!